util/types/src/core/reward.rs

```diff
@@ -1,52 +1,83 @@
 use crate::{core::Capacity, packed::Byte32};
 
-/// TODO(doc): @yangby-cryptape
+/// Details of miner rewards issued by block cellbase transaction.
+///
+/// # References:
+/// - [Token Issuance](https://github.com/nervosnetwork/rfcs/blob/v2020.01.15/rfcs/0015-ckb-cryptoeconomics/0015-ckb-cryptoeconomics.md#token-issuance)
+/// - [Miner Compensation](https://github.com/nervosnetwork/rfcs/blob/v2020.01.15/rfcs/0015-ckb-cryptoeconomics/0015-ckb-cryptoeconomics.md#miner-compensation)
+/// - [Paying for Transaction Fees](https://github.com/nervosnetwork/rfcs/blob/v2020.01.15/rfcs/0015-ckb-cryptoeconomics/0015-ckb-cryptoeconomics.md#paying-for-transaction-fees)
+/// - [`RewardCalculator::txs_fee(..)`](../../ckb_reward_calculator/struct.RewardCalculator.html#method.txs_fees)
+/// - [Collecting State Rent with Secondary Issuance and the NervosDAO](https://github.com/nervosnetwork/rfcs/blob/v2020.01.15/rfcs/0015-ckb-cryptoeconomics/0015-ckb-cryptoeconomics.md#collecting-state-rent-with-secondary-issuance-and-the-nervosdao)
 #[derive(Debug, Default)]
 pub struct BlockReward {
-    /// TODO(doc): @yangby-cryptape
+    /// The total block reward.
     pub total: Capacity,
-    /// TODO(doc): @yangby-cryptape
+    /// The primary block reward.
     pub primary: Capacity,
-    /// TODO(doc): @yangby-cryptape
+    /// The secondary block reward.
+    ///
+    /// # Notice
+    /// Only 60% of the secondary issuance goes to the miners, 35% of the issuance goes to the NervosDAO
+    /// and the last 5% are kept liquid.
```

Comment:
Misleading information. The ratio is calculated dynamically.

* Miner ratio = total occupied CKB / total CKB
* DAO ratio = locked CKB / total CKB
* Others are stored in the header and may be moved to community governed contract in the future.

Reply:
I used the description in RFC-0015 to fix the this incorrect comment.